server/models/Collective.ts

```diff
@@ -3455,9 +3463,19 @@ Collective.init(
       },
     },
 
-    locationName: DataTypes.STRING,
+    locationName: { 
+      type: DataTypes.STRING,
+      validate: {
+        len: [0,5000] // Just to prevent people typing too much in here.
+      },
+    },
 
-    address: DataTypes.STRING,
+    address: { 
+      type: DataTypes.STRING,
+      validate: {
+        len: [0,5000] // Just to prevent people typing too much in here.
+      },
+    },
```

Comment:
Rolling back the changes since these fields are being removed in another PR

```suggestion
    address: DataTypes.STRING,
```